modules/geo-layers/src/tile-3d-layer/get-frame-state.js

```diff
@@ -80,15 +80,10 @@ function commonSpacePlanesToWGS84(viewport) {
 
     const cartesianPos = Ellipsoid.WGS84.cartographicToCartesian(cartographicPos, new Vector3());
 
-    scratchPlane.normal
-      .copy(cartesianPos)
-      .subtract(viewportCenterCartesian)
-      .scale(-1) // Want the normal to point into the frustum since that's what culling expects
-      .normalize();
-    scratchPlane.distance = Math.abs(scratchPlane.normal.dot(cartesianPos));
```

Comment:
This was the issue - `distance` should be `-scratchPlane .normal.dot(cartesianPos)`